service-worker/basic/demo.js

```diff
@@ -0,0 +1,11 @@
+
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register('service-worker.js');
+}
+
+document.querySelector('#show').addEventListener('click', () => {
```

Comment:
My personal preference is very much for `()`, which matches the style used [elsewhere in this repo](https://github.com/GoogleChrome/samples/blob/gh-pages/arrows-es6/demo.js).